cjio/__init__.py

```diff
@@ -1 +1,18 @@
-__version__ = '0.9.0'
\ No newline at end of file
+__version__ = "0.9.0"
+import importlib
+
+
+loader = importlib.util.find_spec("triangle")
```

Comment:
The imports are all checked in this file and in the test of the software the consts are imported from here.